packages/react/src/_popover/Popover.tsx

```diff
@@ -28,11 +31,12 @@ export const Popover = forwardRefWithAs<'div', PopoverProps>(function Popover(
 			borderColor="muted"
 			rounded
 			css={{
-				position: 'relative',
-				overflowY: 'auto',
 				boxShadow: tokens.shadow.lg,
-				zIndex: tokens.zIndex.popover,
+				minHeight: 'min-content',
```

Comment:
Should this have been removed?

Reply:
I just looked into this. It doesn't make a difference to the above/below positioning that floating-ui is picking. I'll explore this a bit further.

- [ ] Explore shift in floating-ui